entproto/cmd/protoc-gen-entgrpc/template/enums.tmpl

```diff
@@ -0,0 +1,23 @@
+{{ define "enums" }}
+    {{- $root := . -}}
+    {{ range .FieldMap.Enums }}
+        {{- $enumType := .PbFieldDescriptor.GetEnumType -}}
+        {{- $enumName := print $root.EntType.Name "_" $enumType.GetName -}}
+        {{- $pbEnumIdent := $root.File.GoImportPath.Ident $enumName   -}}
+        {{- $entLcase := snake $root.EntType.Name -}}
+        {{- $entEnumIdent := entIdent $entLcase .PbStructField -}}
+        func toProto{{ $pbEnumIdent.GoName }} (e {{ $entEnumIdent | ident }}) {{ $pbEnumIdent | ident }} {
```

Comment:
```suggestion
        func toProto{{ $pbEnumIdent.GoName }} (e {{ ident $entEnumIdent }}) {{ ident $pbEnumIdent }} {
```

Don't need to pipe when you have one func. It also makes the code consistent with other parts.

Reply:
🤦